src/main/java/org/jitsi/videobridge/cc/BandwidthProbing.java

```diff
@@ -32,16 +34,45 @@
     extends PeriodicRunnable
 {
     /**
+     * The system property name that holds a boolean that determines whether or
+     * not to activate the RTX bandwidth probing mechanism that implements
+     * stream protection.
+     */
+    public static final String
+        DISABLE_RTX_PROBING_PNAME = "org.jitsi.videobridge.DISABLE_RTX_PROBING";
```

Comment:
yeah, i think it'd be nice to try and avoid the double-negative thing.  but ok, we can hold off on this for this pr.